actions/container-build_test_release/docker-action/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG btag
```

Comment:
Could be better to use `base_tag` to be more informative and consistent with other args.

Reply:
> Could be better to use `base_tag` to be more informative and consistent with other args.

I can change this to `base_tag`. Thanks for pointing this out!

Reply:
`btag` changed to `base_tag`.